src/azul/service/manifest_service.py

```diff
@@ -815,15 +824,39 @@ def manifest_config(self) -> ManifestConfig:
                               combine_script='return new ArrayList(params._agg.fields)',
                               reduce_script=reduce_script)
         es_search = es_search.extra(size=0)
-        response = es_search.execute()
-        assert len(response.hits) == 0
+        try:
+            response = es_search.execute()
+        # If timeout is exceeded the request is partitioned to reduce load on ES
+        except ConnectionTimeout:
```

Comment:
Might be work putting  log.warning here saying that there was a timeout that occurred.